letsencrypt/README.rst

```diff
@@ -88,7 +89,21 @@ You'll also need a matching sudo configuration, like::
 
     your_odoo_user ALL = NOPASSWD: /usr/sbin/service nginx reload
 
-Further, if you force users to https, you'll need something like for nginx::
+The line above can be added to /etc/sudoers through the visudo command, or
+even better be put in a separate file in the /etc/sudoers.d directory. For
```

Comment:
add '...if your distribution supports it' (`sudoers.d` is debian specific to my knowledge)